app/menu.js

```diff
@@ -61,6 +65,56 @@ module.exports = ({createWindow, updatePlugins}) => {
     ]
   };
 
+  const otherShellsMenu = {
+    label: 'Other Shells',
+    submenu: []
+  };
+
+  for (const shellName of Object.keys(shells)) {
```

Comment:
I would save `Object.keys(shells)` as a const since you're using it twice in this file? 🤔